jcapi_test.go

```diff
@@ -498,12 +498,17 @@ func TestRadiusServer(t *testing.T) {
 		t.Fatalf("Could not get all the RADIUS servers, err='%s'", err.Error())
 	}
 
-	if radservers[0].ToString() != rs1.ToString() {
-		t.Fatalf("Systems[0]='%s' - rs1='%s' - string compare failed", radservers[0].ToString(), rs1.ToString())
-	}
-
-	if radservers[1].ToString() != rs3.ToString() {
-		t.Fatalf("Systems[1]='%s' - rs3='%s' - string compare failed", radservers[1].ToString(), rs3.ToString())
+	for _, radServer := range radservers {
+		switch radServer.Id {
+		case rs1.Id:
+			if radServer.ToString() != rs1.ToString() {
+				t.Fatalf("radServer='%s' - rs1='%s' - string compare failed", radServer.ToString(), rs1.ToString())
+			}
+		case rs3.Id:
+			if radServer.ToString() != rs3.ToString() {
+				t.Fatalf("radServer='%s' - rs3='%s' - string compare failed", radServer.ToString(), rs3.ToString())
+			}
+		}
```

Comment:
This might be cleaned up by first generating actual and expected string arrays of rad servers, then deep reflecting they're equal. Alternatively, even if one doesn't want to use goconvey's (our current TDD/BDD framework of choice) full syntax, they can still pull in the underlying assertion helpers they use and use those: https://github.com/smartystreets/assertions/blob/master/equality.go#L139; then you could write something like:
```
message := assetions.ShouldResemble(radserves, []RadServer{r1, r3})
if message != "" {
  t.Fatal(message)
}
```